sqlite-migrations/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
+apply plugin: 'kotlin'
+
+targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_7
+
+configurations {
+  bundled
```

Comment:
this configuration can be removed. The gradle plugin is the only thing that should be bundling